### PR TITLE
user can choose whether to sync user settings of PackageSync

### PR DESCRIPTION
**Pull request by EthanZhong02**
Hi, I add a little bit modification to it, which can make user choose whether to include **PackageSync.sublime-settings** in the sync.

### CODE REVIEW

File: PackageSync.sublime-settings [hunk 1]
```diff
@@ -21,7 +21,9 @@
 		"Package Control.ca-list",
 		"Package Control.ca-bundle",
 		"Package Control.system-ca-bundle",
-		"*.sublime-package"
+		"*.sublime-package",
+		"PackageSync.sublime-settings",
```
Comment (utkarsh9891) [inline PackageSync.sublime-settings]: Preferably add a different configuration/flag that would define if the packages are to be included e.g. `sync_package_sync_settings = true`

File: PackageSync.sublime-settings [hunk 2]
```diff
@@ -21,7 +21,9 @@
...
+		"PackageSync.last-run"
```
Comment (utkarsh9891) [inline PackageSync.sublime-settings]: `PackageSync.last-run` should never be synced as it stores the sync logs. Syncing that would mess up the entire logic & might cause infinite recursion across devices.

File: package_sync_helpers/tools.py [hunk 3]
```diff
@@ -20,7 +20,7 @@ def get_psync_settings():
         "list_backup_path": s.get("list_backup_path", ""),
         "zip_backup_path": s.get("zip_backup_path", ""),
         "folder_backup_path": s.get("folder_backup_path", ""),
-        "ignore_files": s.get("ignore_files", []) + ["PackageSync.sublime-settings", "PackageSync.last-run"],
+        "ignore_files": s.get("ignore_files", []),
```
Comment (utkarsh9891) [inline package_sync_helpers/tools.py]: Use the flag to define if PackageSync settings are to be synced.
Still, It is not advisable to use this feature. Would think about the repercussions of the same before merging.

**Review by utkarsh9891 — CHANGES_REQUESTED** [review]

File: PackageSync.sublime-settings (hunk 2 above)
Comment (EthanZhong02) [reply PackageSync.sublime-settings]: yeah, I didn't notice the default all files with  suffix `.last-run` will be ignored by default.

**Comment by EthanZhong02** [comment]
I have added new changes which added  `sync_package_sync_settings` option to choose whether to synchronize user setting, but there are conflicts to the `master` branch.

File: PackageSync.sublime-settings [hunk 4]
```diff
@@ -24,6 +24,9 @@
 		"*.sublime-package"
 	],
 
+	// Toggle to determine whether to synchronize user setting for this package (`PackageSync.sublime-settings`).
+	"sync_package_sync_settings": false,
```
Comment (utkarsh9891) [inline PackageSync.sublime-settings]: Please add the following to the comment as well
`Caution: Use this feature very cautiously. This overrides your PackageSync settings across the synced devices. If you wish to keep different sync folder paths across the devices, DO NOT set this as True`


Also, please add the configuration and the corresponding comments to the README.md file.

**Review by utkarsh9891 — CHANGES_REQUESTED** [review]
Update the PR with conflicts resolved.
Please add a messages file with version 2.1.1.txt and also add corresponding entry in messages.json so that this can be pushed as a minor-minor release.

**Comment by EthanZhong02** [comment]
new commits added

File: messages.json [hunk 5]
```diff
@@ -6,6 +6,7 @@
 	"1.3.1": "messages/1.3.1.txt",
 	"1.3.2": "messages/1.3.2.txt",
 	"1.4.0": "messages/1.4.0.txt",
-  "2.0.0": "messages/2.0.0.txt",
-	"2.1.0": "messages/2.1.0.txt"
+	"2.0.0": "messages/2.0.0.txt",
+  "2.1.0": "messages/2.1.0.txt",
```
Comment (utkarsh9891) [inline messages.json]: Add indentation

File: messages/2.1.1.txt [hunk 6]
```diff
@@ -0,0 +1,7 @@
+=============
+Release 2.1.1
+=============
+
+PackageSync now can choose to synchronize user setting or not.
```
Comment (utkarsh9891) [inline messages/2.1.1.txt]: Typos. Set to `You now have support to sync your PackageSync settings across devices as well. Do use this with caution though!!!`

**Review by utkarsh9891 — CHANGES_REQUESTED** [review]

**Review by utkarshzomato — APPROVED** [review]

**Review by utkarsh9891 — APPROVED** [review]

**Pull request merged**